src/App.jsx

```diff
@@ -1,17 +1,19 @@
-import { useState } from "react";
-import "./App.css";
+import useSWR from 'swr';
+import './App.css';
 
 function App() {
-  const url = "https://httpstat.us/200?sleep=2000";
-  const headers = { Accept: "application/json" };
+  const url = 'https://httpstat.us/200?sleep=2000';
+  const headers = { Accept: 'application/json' };
 
-  const [status, setStatus] = useState("");
+  const fetcher = (url) =>
+    fetch(url, { headers })
+      .then((res) => res.json())
+      .then((json) => json.description);
+  const { data, error, isLoading } = useSWR(url, fetcher);
+  if (error) return <div>Failed to load.</div>;
+  if (isLoading) return <div>Loading...</div>;
```

Comment:
[nits]
この辺は改行入れてあげると見やすそうですね！

```suggestion
  const fetcher = (url) =>
    fetch(url, { headers })
      .then((res) => res.json())
      .then((json) => json.description);
      
  const { data, error, isLoading } = useSWR(url, fetcher);
  
  if (error) return <div>Failed to load.</div>;
  if (isLoading) return <div>Loading...</div>;
```